services/metadata/src/metadata/routes/templates.clj

```diff
@@ -24,13 +24,27 @@
       :description "This endpoint returns the details of a single metadata attribute."
       (ok (templates/view-attribute attr-id)))
 
-    (GET* "/:template-id" []
+    (context* "/:template-id" []
       :path-params [template-id :- TemplateIdPathParam]
-      :query [params StandardUserQueryParams]
-      :return MetadataTemplate
-      :summary "View a Metadata Template"
-      :description "This endpoint returns the details of a single metadata template."
-      (ok (templates/view-template template-id)))))
+
+      (GET* "/" []
+        :query [params StandardUserQueryParams]
+        :return MetadataTemplate
+        :summary "View a Metadata Template"
+        :description "This endpoint returns the details of a single metadata template."
+        (ok (templates/view-template template-id)))
+
+      (GET* "/blank-csv" []
+        :query [{:keys [attachment]} CSVDownloadQueryParams]
+        :summary "Get a blank CSV template file for a metadata template."
+        :description "This endpoint returns a CSV file suitable for a specific template, ready to be filled in with specific values. It's intended to be downloaded to be filled out by the user, then reuploaded for use with the bulk metadata endpoints."
+        (templates/csv-download-resp attachment "metadata.csv" (templates/view-template-csv template-id)))
```

Comment:
this filename, and the other, are generic like this because we don't really sanitize the template names at all so I didn't want to use them, and the UUIDs are pretty much internal/meaningless to a user. if it should be a requirement that the suggested download filename include the name of the template, we could look into filename sanitization. The user can, of course, override this name anyway.